stdlib/public/core/DoubleWidth.swift.gyb

```diff
@@ -52,11 +53,25 @@ public struct DoubleWidth<Base : FixedWidthInteger> : _ExpressibleByBuiltinInteg
   }
 }
 
-extension DoubleWidth: Comparable {
+extension DoubleWidth : CustomStringConvertible {
+  @_inlineable // FIXME(sil-serialize-all)
+  public var description: String {
+    return String(self)
```

Comment:
I agree that this change is better, but the performance can degrade pretty quickly since division is slow, esp. if `DoubleWidth`s are nested. What would you think about using hexadecimal instead?

Reply:
That sounds right. If somebody does run into trouble they can just use `String(x, radix: 16)` instead.